docs/ops/sort/TopK_11.md

```diff
@@ -77,12 +77,39 @@ Sorting and minimum/maximum are controlled by `sort` and `mode` attributes with
   * *sort*=`value`, *mode*=`max`, *stable*=`true`  - descending by value, relative order of equal elements guaranteed to be maintained
   * *sort*=`value`, *mode*=`min`, *stable*=`false` - ascending by value, relative order of equal elements not guaranteed to be maintained
   * *sort*=`value`, *mode*=`min`, *stable*=`true`  - ascending by value, relative order of equal elements guaranteed to be maintained
-  * *sort*=`index`, *mode*=`max` - descending by index
-  * *sort*=`index`, *mode*=`min` - ascending by index
+  * *sort*=`index`, *mode*=`max`, *stable*=`false` - descending by index, relative order of equal elements not guaranteed to be maintained
```

Comment:
Please revert all the "descending by index" to be "ascending by index", as we've discussed in https://github.com/openvinotoolkit/openvino/pull/15583#discussion_r1133727732.

Reply:
Done, please have a look if the clarification in this commit https://github.com/openvinotoolkit/openvino/pull/16811/commits/a8e3ddad75a03d71c29dd278b0d1a2b4e56477e1 makes sense.